data/strings/strings.txt

```diff
@@ -20153,93 +20153,95 @@
 
   [routing_add_start_point]
     tags = android,ios
-    en = Add a starting point to plan a route
-    af = Voeg ’n beginpunt toe om ’n roete te beplan
-    ar = أضف نقطة البداية لتخطيط المسار
-    az = Marşrutu planlaşdırmaq üçün başlanğıc nöqtəsi əlavə edin
-    be = Дадайце пункт адпраўлення, каб пракласці маршрут
-    bg = Добавяне на начална точка за планиране на маршрут
-    ca = Afegiu un punt inical per a planificar una ruta
-    cs = Zadejte výchozí bod pro plánování trasy
-    da = Tilføj startpunkt for at planlægge en rute
-    de = Fügen Sie einen Startpunkt hinzu, um eine Route zu planen
-    el = Προσθέστε αφετηρία για να σχεδιάσετε μια διαδρομή
-    es = Buscar un origen para el plan de ruta
-    et = Marsruudi planeerimiseks lisa lähtepunkt
-    eu = Gehitu abiapuntu bat ibilbide bat antolatzeko
-    fa = یک نقطه شروع برای برنامه ریزی یک مسیر اضافه کنید
-    fi = Lisää alkupiste reitin suunnittelua varten
-    fr = Ajouter un point de départ pour planifier un itinéraire
-    he = יש להוסיף נקודת התחלה כדי לתכנן מסלול
-    hi = मार्ग की योजना बनाने के लिए एक प्रारंभिक बिंदु जोड़ें
-    hu = Adjon hozzá kiindulási pontot az útvonal megtervezéséhez
-    id = Tambahkan titik awal untuk merencanakan rute
-    it = Aggiungi un punto di partenza per pianificare un percorso
-    ja = ルートを計画するには出発地を追加してください
-    ko = 경로 계획을 세우기 위한 시작 지점을 추가하세요
-    lt = Pridėkite pradžios tašką, kad planuotumėte maršrutą
-    mr = मार्ग नियोजित करण्यासाठी प्रारंभ बिंदू जोडा
-    nb = Angi startpunkt for å planlegge rute
-    nl = Voeg beginpunt toe om een route te plannen
-    pl = Aby zaplanować trasę, dodaj punkt początkowy
-    pt = Adicionar ponto de partida para planear uma rota
-    pt-BR = Adicionar ponto de partida para planejar uma rota
-    ro = Adaugă un punct de plecare pentru a planifica un traseu
-    ru = Добавьте стартовую точку, чтобы построить маршрут
-    sk = Pridaním počiatočného bodu začnite plánovať trasu
-    sv = Lägg till startpunkt för att planera en rutt
-    sw = Ongeza eneo la kuanzia ili kupanga njia ya kufuata
-    th = เพิ่มจุดเริ่มต้นเพื่อวางแผนเส้นทาง
-    tr = Rotayı planlamak için bir başlangıç noktası ekleyin
-    uk = Вкажіть початкову точку для побудови маршруту
-    vi = Bổ sung điểm bắt đầu để lập kế hoạch lộ trình
-    zh-Hans = 添加起点以规划路线
-    zh-Hant = 新增起點以計劃路線
+    comment = User selected the destination by pressing Route To, but the current position is unknown. User needs to select a starting point of a route using search or by tapping on the map and then pressing "Route From".
```

Comment:
@organicmaps/translations can you please help with proof-reading this translation in your language (or provide a clearer version)?